packages/terra-graphs-docs/CHANGELOG.md

```diff
@@ -5,3 +5,6 @@
 * Added 
   * Created this separate package for terra-dev-site and documentation.
   * Added data objects and examples for Line, Bar and Paired result graphs.
+
+* Changed
+  * Scss file refactoring. 
```

Comment:
nit: Maybe we don't need this `Changed` entry since terra-graphs-docs hasn't been released yet...unless we want to keep a history of all the changes that go into it.

Reply:
It just requires any kind of change in the changelog, not necessarily a new entry. We can leave it as is and make any necessarily updates during release time if needed.

Reply:
Removed [here](https://github.com/cerner/terra-graphs/pull/122/commits/d7de03a2d8a46988ace790c2ae4c36ca3dd16702#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1L9).

Reply:
You can update the danger file in this project to ignore packages that have 'ignoreChanges' set on them in the lerna config:
For example here's the terra-application danger file.
https://github.com/cerner/terra-application/blob/terra-application-v2/dangerfile.js#L8-L10